src/main/scala/it/pagopa/interop/backendforfrontend/service/types/CatalogProcessServiceTypes.scala

```diff
@@ -96,7 +96,15 @@ object CatalogProcessServiceTypes {
         name = es.name,
         description = es.description,
         technology = es.technology.toProcess,
-        mode = es.mode.toProcess
+        mode = es.mode.toProcess,
+        descriptor = CatalogProcess.DescriptorSeedForEServiceCreation(
+          description = None,
+          audience = Seq(),
+          voucherLifespan = 1,
```

Comment:
constraint in process: min value 60

Reply:
Thanks! I think I got confused between minutes and seconds.